LOAN/Loans/LoanBasicTerms.rdf

```diff
@@ -306,7 +306,7 @@
 	
 	<owl:ObjectProperty rdf:about="&fibo-loan-loan-loan;spread">
 		<rdfs:label xml:lang="en">spread</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-loan-loan;LoanFloatingInterestPaymentTerms"/>
+		<rdfs:domain rdf:resource="&fibo-fbc-dae-dbt;InterestPaymentTerms"/>
```

Comment:
No idea where this came from or why it was included in this PR, which would not otherwise have even touched this ontology.